ext/node/polyfills/_http_outgoing.ts

```diff
@@ -820,13 +820,13 @@ Object.defineProperty(OutgoingMessage.prototype, "_headerNames", {
   ),
 });
 
-export const validateHeaderName = hideStackFrames((name) => {
+export const validateHeaderName = hideStackFrames((name:string, label?: string): void => {
```

Comment:
**nitpick (blocking):** It seems like the formatting here is not correct, `name:string` is missing a space after the colon. Did you run `./tools/format.js`?